wordpress-mobile/WordPress-iOS
# Migrates ContextManager to Swift

Implements a new `CoreDataManager` type that is the Swift version of `ContextManager`. If the feature flag is enabled, `ContextManager` is replaced by `CoreDataManager`.

The same way, the correct mocks are applied during the tests based on the flag.

To test:

- try as many use cases as possible and verify that the app behaves as expected (I know, it's pretty broad..)

PR submission checklist:

- [x] ~~I have considered adding accessibility improvements for my changes.~~ This PR does not contain UI changes.
- [x] ~~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~~ Not user-facing changes.
